File: split/newskipparam.pas
function newskipparam(n: smallnumber): halfword;
var
  p: halfword;
begin
  tempptr := newspec({224:}
eqtb[2882 + n].hh.rh{:224}
);
  p := newglue(tempptr);
  mem[tempptr].hh.rh := 0;
  mem[p].hh.b1 := n + 1;
  newskipparam := p;
end;

File: cpp/newskipparam.cpp
#include "newskipparam.h"
#include "newspec.h"
#include "newglue.h"

halfword newskipparam(smallnumber n)
{
	tempptr = newspec(glue_par(n));
	auto p = newglue(tempptr);
	link(tempptr) = 0;
	subtype(p) = n+1;
	return p;
}
